### PR TITLE
Comments in index js file + refactor code

### PR DESCRIPTION
**Pull request by adsingh14**
Added some comments and refactor the code

Issue:

### CODE REVIEW

File: assets/js/index.js [hunk 1]
```diff
@@ -186,36 +166,80 @@ $(document).ready(function () {
...
-    if (w < 768) {
-      toggleSidebar();
+  // Download Button button
+  $(
+    "#downloadGithubRawHeader, #downloadGithubRawButton, #downloadGithubRawHowTo"
+  ).on("click", downloadGithubCssRaw);
+
+  function downloadGithubCssRaw() {
+    var link =
+      "https://cdn.statically.io/gh/shahednasser/sbuttons/c135f5f7/dist/sbuttons.min.css";
```
Comment (shahednasser) [inline assets/js/index.js:221]: change link from "shahednasser/sbuttons" to "sButtons/sbuttons" as we changed it to organization

**Review by shahednasser — CHANGES_REQUESTED** [review]

**Review by adsingh14 — COMMENTED** [review]
Done !

File: assets/js/index.js (hunk 1 above)
Comment (adsingh14) [reply assets/js/index.js]: Done in recent [PR](https://github.com/sButtons/sbuttons/pull/583/commits/2b0f376df56af6a20f2cc3207aa8de4e84a8d511).

**Comment by shahednasser** [comment]
Great thank you!

@all-contributors please add @adsingh14 for documentation

File: assets/js/index.js (hunk 1 above)
Comment (adsingh14) [reply assets/js/index.js]: Is this issue still unresolved? I still got red icon notification.

**Pull request merged**